src/routes/GroupDetail/GroupDetail.connector.js

```diff
@@ -49,7 +49,7 @@ export function mapDispatchToProps (dispatch, props) {
   return {
     addSkill: (name) => dispatch(addSkill(name)),
     removeSkill: (skillId) => dispatch(removeSkill(skillId)),
-    fetchGroup: () => dispatch(fetchGroupDetails({ slug, withWidgets: true })),
+    fetchGroup: (currentUser) => dispatch(fetchGroupDetails({ slug, withWidgets: true, withPrerequisites: !!currentUser })),
```

Comment:
you could also handle this on the backend entirely, but this is ok

Reply:
How so? The frontend was blowing up because it was making queries that expected a valid `currentUser` but that isn't assured in this context, so I just needed to adjust this part of the query to be included or not depending on the presence of a currentUser

And, didn't want to make a backend change if I didn't need to.